### PR TITLE
[docs] Share Cloudprober's journey through an About page.

### PR DESCRIPTION
**Pull request by manugarg**

### CODE REVIEW

File: docs/content/docs/about/index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+and back-and-forth[^1].
+
+[^1]:
+    A customer-reported infrastructure issue is much harder to debug than an
+    issue discovered by your own observability stack.
+
+Problem was that Google's existing observability tools didn't work for the
+external Cloud. We needed to build things from ground up. As probers[^2] are
+pretty much the foundation of the reliable observability at Google, we decided
+to prioritize the developement of a prober for Cloud. That's how the journey of
```
Comment (robpickerill) [inline docs/content/docs/about/index.md:22]: a minor typo here Manu, `developement` -> `development`

File: docs/content/docs/about/index.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+Cloud teams started using it internally, and open-source readiness commitment
+made sure we kept our interfaces clean.
+
+## Built for Scale
+
+Cloudprober was built to probe 100s of 1000s of endponts (IPs and HTTP URLs),
+while keeping resources, and more importantly, management overhead very very
+low. That's the reason Cloudprober tries to be scrappy with resources, maximizes
+resources utilization relying heavily on Go concurrency, supports probing many
+many targets in parallel at a very high freuency, miminimizes the need of
```
Comment (robpickerill) [inline docs/content/docs/about/index.md:45]: a minor typo: `.. frequency, minimizes ..`

**Comment by robpickerill** [comment]
Thanks for sharing the background of Cloudprober here, a good read.

File: docs/content/docs/about/index.md (hunk 2 above)
Comment (manugarg) [reply docs/content/docs/about/index.md:45]: Thank you!

**Pull request merged**